README.md

```diff
@@ -46,6 +46,12 @@ If errors are not shown in pull request, but mypy fails, run mypy locally with t
 python -m mypy.
 ```
 
+## Mypy
+Run all tests:
+```
+pytest .
+```
+
```

Comment:
Titlen er 'Mypy' men det om handler pytest. 
Jeg tror også vi skal bruge `python -m pytest` for at det virker, men er ikke sikker.